### PR TITLE
fix(VSlider): add missing prepend and append icon click events

### LINKED ISSUE #16071: [Bug Report][3.0.1] `v-slider` append and prepend event is not working
### Environment
**Vuetify Version:** 3.0.1
**Vue Version:** 3.2.44
**Browsers:** Chrome 107.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Check codepen

### Expected Behavior
Append and prepend events should be emitted when clicking on `prepend-icon` and `append-icon`

### Actual Behavior
Not Working

### Reproduction Link
[https://codepen.io/Alokdev/pen/ZERePMB?editors=101](https://codepen.io/Alokdev/pen/ZERePMB?editors=101)

### PR DESCRIPTION
**Pull request by vexleet**
## Motivation and Context
There were props for adding click events for the append and prepend icons in the v slider but they weren't used in the component which led to not being able to have events for the icons, which would fix #16071

## How Has This Been Tested?
I created a unit test using cypress which adds a slider with 2 icons and putting events on them, after that I used cypress spies to check if the button events worked when both icons were clicked.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by nekosaur** [comment]
@KaelWD we're filtering out event props here https://github.com/vuetifyjs/vuetify/blob/f627eb1726f1e195b11a984bedc79772a682b2e8/packages/vuetify/src/components/VInput/VInput.tsx#L191 which is the root cause of this issue. What's the reason we're doing this?

**Comment by KaelWD** [comment]
I think because VTextField does some other shit first then waits a tick before re-emitting clear:
https://github.com/vuetifyjs/vuetify/blob/f6ee9a33af5bc0b312158738c97dd226837cd45d/packages/vuetify/src/components/VTextField/VTextField.tsx#L126-L136
We could probably do something similar to line 145 though and just 
`const [{ 'onClick:clear': _, ...fieldProps }] = filterFieldProps(props)`

**Comment by johnleider** [comment]
Is that the advised direction @KaelWD? To add exceptions for those events?

**Comment by johnleider** [comment]
@KaelWD @nekosaur any updates?

**Comment by johnleider** [comment]
Possibly related: #15904, #16945, 133b4172550ec7d5ddaae19085761f31c5ce3548

**Comment by yuwu9145** [comment]
> I think because VTextField does some other stuff first then waits a tick before re-emitting clear:
> 
> https://github.com/vuetifyjs/vuetify/blob/f6ee9a33af5bc0b312158738c97dd226837cd45d/packages/vuetify/src/components/VTextField/VTextField.tsx#L126-L136
> 
> 
> If clear was part of fieldProps it'd happen twice
> We could probably do something similar to line 145 though and just
> `const [{ 'onClick:clear': _, ...fieldProps }] = filterFieldProps(props)`

VField is emitting "prependInner" rather than "prepend". We should focus on `filterInputProps`. 

Removing `.filter(k => !isOn(k))` makes sense, however, would this be a breaking change?

File: packages/vuetify/src/components/VInput/VInput.tsx [hunk 1]
```diff
@@ -190,6 +190,6 @@ export const VInput = genericComponent<VInputSlots>()({
 export type VInput = InstanceType<typeof VInput>
 
 export function filterInputProps (props: Record<string, unknown>) {
```
Comment (KaelWD) [inline packages/vuetify/src/components/VInput/VInput.tsx:192]: This function can be removed and replaced with `VInput.filterProps` if it isn't handling events differently anymore.

Comment (yuwu9145) [reply packages/vuetify/src/components/VInput/VInput.tsx]: nice ! 🎉

**Review by johnleider — APPROVED** [review]

**Pull request merged**